src/dispatch/function.py

```diff
@@ -256,9 +256,9 @@ def primitive_func(input: Input) -> Output:
             return OneShotScheduler(func).run(input)
 
         primitive_func.__qualname__ = f"{name}_primitive"
-        primitive_func = durable(primitive_func)
+        durable_primitive_func = durable(primitive_func)
```

Comment:
mypy complained here that the type of `primitive_func` didn't match after being wrapped by `durable`.